snitun/multiplexer/core.py

```diff
@@ -32,6 +33,8 @@
 
 PEER_TCP_TIMEOUT = 90
 
+HEADER_STRUCT = struct.Struct(">16sBI11s")
```

Comment:
I think the format string needs a code comment explaining it.

Reply:
added a comment to explain the header